std/process.d

```diff
@@ -132,6 +132,7 @@ version (Posix)
 {
     version (OSX)
     {
+        import std.internal.phobosinit; // needed to make sure the function gets called
```

Comment:
Yes, I think that's why default hello world goes up in space so much. Ugly as it is, we need an individual internal module for each module that has cycles.